hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -160,7 +160,7 @@ public class HiveSyncConfig extends HoodieSyncConfig {
 
   public static final ConfigProperty<String> HIVE_SUPPORT_TIMESTAMP_TYPE = ConfigProperty
       .key("hoodie.datasource.hive_sync.support_timestamp")
-      .defaultValue("false")
+      .defaultValue("true")
       .withDocumentation("‘INT64’ with original type TIMESTAMP_MICROS is converted to hive ‘timestamp’ type. "
           + "Disabled by default for backward compatibility.");
```

Comment:
`withDocumentation`‘s content is  need change also?

Reply:
@XuQianJin-Stars 
Can this shows whether we can clearly explain the PR?

'INT64' with original type TIMESTAMP_MICROS is converted to hive 'timestamp' type. From 0.12.0, 'timestamp' type will be supported and also can be disabled by this variable. Previous versions keep being disabled by default.

Reply:
> @XuQianJin-Stars Can this shows whether we can clearly explain the PR?
> 
> 'INT64' with original type TIMESTAMP_MICROS is converted to hive 'timestamp' type. From 0.12.0, 'timestamp' type will be supported and also can be disabled by this variable. Previous versions keep being disabled by default.

In `deprecatedAfter` method write version 0.12.0 and change withDocumentation‘s content?

Reply:
> > @XuQianJin-Stars Can this shows whether we can clearly explain the PR?
> > 'INT64' with original type TIMESTAMP_MICROS is converted to hive 'timestamp' type. From 0.12.0, 'timestamp' type will be supported and also can be disabled by this variable. Previous versions keep being disabled by default.
> 
> In `deprecatedAfter` method write version 0.12.0 and change withDocumentation‘s content?

Is this right? 
```java
  public static final ConfigProperty<String> HIVE_SUPPORT_TIMESTAMP_TYPE = ConfigProperty
      .key("hoodie.datasource.hive_sync.support_timestamp")
      .defaultValue("true")
      .deprecatedAfter("0.12.0")
      .withDocumentation("'INT64' with original type TIMESTAMP_MICROS is converted to hive 'timestamp' type. "
          + "From 0.12.0, 'timestamp' type will be supported and also can be disabled by this variable. "
          + "Previous versions keep being disabled by default.");
```
If there's no problem, I'll change all the other descriptions.